tripy/nvtripy/frontend/module/groupnorm.py

```diff
@@ -35,6 +37,10 @@ class GroupNorm(Module):
     :math:`\text{GroupNorm}(x) = \Large \frac{x - \bar{x}}{ \sqrt{\sigma^2 + \epsilon}} \normalsize * \gamma + \beta`
 
     where :math:`\bar{x}` is the mean and :math:`\sigma^2` is the variance.
+
+    The input should have shape :math:`[N, C, D1, ...]` where :math:`N` is the batch size, :math:`C` is the number of channels, and :math:`D1, ...` are the feature dimensions.
+
+    Note that the current implementation uses InstanceNorm as a workaround to implement this operation due to TRT API compatibility issues.
```

Comment:
Unless this is affects the functionality, I would consider this an implementation detail and not include it in the docstring. 
```suggestion
```